pkg/utils/utils.go

```diff
@@ -100,6 +101,28 @@ func (u *utils) CreatePodOnNode(nodeName, namespace, image string) (*corev1.Pod,
 	return pod, nil
 }
 
+func (u *utils) CreatePodOnNode(nodeName, namespace, image string) (*corev1.Pod, error) {
+	pod := getPodDefinition(nodeName, namespace, image, []string{})
+	return u.createPod(namespace, pod)
+}
+
+func (u *utils) CreatePodOnNodeWithCommand(nodeName, namespace, image string, command []string) (*corev1.Pod, string, error) {
```

Comment:
The name of a func is CreatePodOnNodeWithCommand while it creates a pod and returns its log, it's not describing well what this func does, and anyway I don't think getting the log should be part of create pod func